pkg/operator/configobservation/images/observe_images.go

```diff
@@ -31,11 +31,6 @@ func ObserveInternalRegistryHostname(genericListers configobserver.Listers, reco
 		}
 	}
 
-	if !listers.ImageConfigSynced() {
-		glog.Warning("images.config.openshift.io not synced")
-		return prevObservedConfig, errs
-	}
-
```

Comment:
couple lines below we have:

```
	if errors.IsNotFound(err) {
		glog.V(2).Infof("images.config.openshift.io/cluster: not found")
		return observedConfig, errs
	}
```

shouldn't this be `prevObservedConfig` ?

Reply:
no...  if you don't have an images cluster config object, then we should return an empty observed config because you have an empty images config.

if we get a different error (ie api error) then we return the prev observed config.